adam_core/coordinates/covariances.py

```diff
@@ -153,6 +153,25 @@ def is_all_nan(self) -> bool:
         """
         return np.all(np.isnan(self.to_matrix()))
 
+    @classmethod
+    def nulls(cls, length: int) -> "CoordinateCovariances":
```

Comment:
This function is helpful when creating a "null" covariances which can be written to and read from Parquet, because https://github.com/apache/arrow/issues/35692 remains an issue.